applications/GeoMechanicsApplication/tests/line_interface_elements/Dirichlet_multi_stage/MaterialParameters.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Yes, that was exactly the intention, but it hadn't been implemented yet, since the branch was lagging behind a bit when you reviewed it. Your suggestion has now been processed. Thank you for reminding me.